Framework/WorkflowAlgorithms/src/EQSANSLoad.cpp

```diff
@@ -737,7 +737,7 @@ void EQSANSLoad::exec() {
   // Modify TOF
   const bool correct_for_flight_path = getProperty("CorrectForFlightPath");
   double wl_min = 0.0;
```

Comment:
```suggestion
  double wl_min;
```
Is there any reason we can't do the same to wl_min and wl_combined_max?

Reply:
Because it is possible that the initial value for `wl_min` may be used at line 834 and `wl_combined_max` at line 835. `cppcheck` seems to do pretty throughout work here.

Reply:
Just to reiterate, it is not possible to remove the initialization from `wl_min` and `wl_combined_max`.